assets/src/components/notificationModals/basicNotificationModal.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { useContext } from "react"
+import { Modal } from "@restart/ui"
+import { setNotification } from "../../state"
+import { StateDispatchContext } from "../../contexts/stateDispatchContext"
+import { OldCloseIcon } from "../../helpers/icon"
+
+const BasicNotificationModal = ({
+  title,
+  body,
+}: {
+  title: string
+  body: string
+}) => {
+  const [, dispatch] = useContext(StateDispatchContext)
+
+  const closeModal = () => {
+    dispatch(setNotification())
+  }
+
+  return (
+    <Modal
+      className="c-modal"
+      show
+      renderBackdrop={(props) => (
+        <div {...props} className="c-modal-backdrop" />
+      )}
+    >
+      <div className="c-basic-notification-modal">
+        {/* TODO: Close button IS tab-able, but not visually given focus */}
+        <button
+          title="Close"
+          className="c-basic-notification-modal__close-button"
+          onClick={closeModal}
+        >
+          <OldCloseIcon />
+        </button>
+        <div className="c-notification__title">{title}</div>
```

Comment:
question: is this intended to be `.c-notification__title`? If this style is apart of the `.c-basic-notification-modal` component, I think we should move this styling into a `.c-basic-notification-modal`, but if the restart ui modal ends up throwing away any of this, I'd be open to making this non-blocking.

Reply:
Oh I didn't realize the other modals did this too :melt:

Reply:
Rethinking this some more, consider this non-blocking for now.

Reply:
Ah I see. I don't see any styles attached to this selector anyway, so I'd be open to just eliminating the className entirely